data/json/items/comestibles/meat_dishes.json

```diff
@@ -661,7 +685,7 @@
     "symbol": "%",
     "quench": 2,
     "calories": 142,
-    "description": "Delicious, lumpy, white soup made of clams and potatoes.  A taste of the lost glory of New England.",
+    "description": "Delicious, lumpy, white soup made of meat and potatoes.  A taste of the lost glory of New England.",
```

Comment:
I'd suggest
"Delicious, lumpy, white soup made of meat (normally clam) and potatoes.  A taste of the lost glory of New England."